src/client/client.test.ts

```diff
@@ -98,34 +96,32 @@ describe('superface client', () => {
   it('caches super.json files correctly', () => {
     const client = new SuperfaceClient();
     expect(client.superJson.document).toEqual(MOCK_SUPERJSON);
-    expect(statSyncMock).toHaveBeenCalledTimes(1);
```

Comment:
Isn't this like saying: const x = 3, expect(x === x).toBe(true)? 🤷

Reply:
https://applifting.slack.com/archives/C01PVQBACB1/p1624269231008500

Reply:
✅